application.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask
 from flask_cors import CORS
 from controllers import *
+from controllers.change_diet_requirement import diet_requirement_change_bp
```

Comment:
- This import should not be necessary, it should be covered by the previous line. 
import * means import all